### PR TITLE
fixes #107 -add a tag list legend

### LINKED ISSUE #107: Add hover description for tags
Each tag should have a hover description:

here are the proposed texts: 
* dates -> hover description: "Last presented on ...."
* tests -> hover description: "has tests"
* specification -> hover description: "has specification text"

### PR DESCRIPTION
**Pull request by codehag**
This adds a details section for explaining the tags in the proposals

### CODE REVIEW

**Comment by codehag** [comment]
@ljharb  your thoughts would be appreciated !

**Comment by ljharb** [comment]
Is there a way to view this PR rendered?

**Comment by codehag** [comment]
yes, via netlify: https://deploy-preview-121--tc39-beta.netlify.com/

you can find the link here:

Screenshot 2019-03-09 at 20 09 08

File: _includes/proposals.html [hunk 1]
```diff
@@ -6,6 +6,33 @@ <h3>Current Candidates for the Specification</h3>
...
+      Each proposal has links out to other resources. The tags identify the following attributes.
+      Below you will find a table with example tags and their meanings.
+      All tags have hover text if you need help getting oriented.
+      </p>
+
+      <strong>Types of tags and what they mean</strong>
+      <ul class="featurelist__legend__tags">
+          <li>Last Presented:
+          <div class="featurelist__item__presented featurelist__item__tag">
+            <a href="./" title="Notes from most recent presentation">December 2018</a>
```
Comment (ljharb) [inline _includes/proposals.html]: I’d probably link these to `#` so they don’t navigate when clicked?

File: _includes/proposals.html [hunk 2]
```diff
@@ -6,6 +6,33 @@ <h3>Current Candidates for the Specification</h3>
...
+          </div>
+          </li>
+          <li>Available Tests:
+          <div class="featurelist__item__tests featurelist__item__tag">
+            <a href="./" title="Patch introducing tests in test262 repository">Test</a>
+          </div>
+          </li>
+          <li>Specification Text:
+        <div class="featurelist__item__spec featurelist__item__tag">
```
Comment (ljharb) [inline _includes/proposals.html]:
```suggestion
            <div class="featurelist__item__spec featurelist__item__tag">
```

File: _includes/proposals.html [hunk 3]
```diff
@@ -6,6 +6,33 @@ <h3>Current Candidates for the Specification</h3>
...
+          <a href="./" title="Read the specification text">Specification</a>
+        </div>
```
Comment (ljharb) [inline _includes/proposals.html]:
```suggestion
            </div>
```

File: _includes/proposals.html [hunk 4]
```diff
@@ -6,6 +6,33 @@ <h3>Current Candidates for the Specification</h3>
...
+          </div>
+          </li>
+          <li>Available Tests:
+          <div class="featurelist__item__tests featurelist__item__tag">
+            <a href="./" title="Patch introducing tests in test262 repository">Test</a>
+          </div>
+          </li>
+          <li>Specification Text:
+        <div class="featurelist__item__spec featurelist__item__tag">
+          <a href="./" title="Read the specification text">Specification</a>
```
Comment (ljharb) [inline _includes/proposals.html]:
```suggestion
              <a href="./" title="Read the specification text">Specification</a>
```

**Review by ljharb — APPROVED** [review]
Awesome, the preview’s great! LGTM, only minor suggestions.

**Pull request merged**